examples/layer_shell.rs

```diff
@@ -0,0 +1,111 @@
+#![allow(warnings)]
```

Comment:
Can we add another example or ad to this example to on demand reconfigure the window?